src/main/resources/data/pieces/313000-02_created_by_instance.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "98ecf23b-f61a-376c-b53b-a5fb3aa70f9f",
+  "caption": "Tutorial Volume 35",
+  "comment": "Special Edition",
+  "format": "Electronic",
+  "locationId": "f34d27c6-a8eb-461b-acd6-5dea81771e70",
```

Comment:
This piece should have itemId since `poLine.eresource.createInventory="Instance, Holding, Item"`